src/data-restriction/DataRestrictionUtils.js

```diff
@@ -86,22 +86,11 @@ export function isPrereleaseStudyTemp (access) {
 // - the user doesnt have access
 export function isPrereleaseStudy (access, studyId, user, permissions) {
```

Comment:
Do we still need the `user` argument here?

Reply:
ooh good point. 

I don't see why we would care if `user` is undefined here, since we aren't using it within the logic. I went ahead and removed it!

Reply:
One thing to watch out for: `@veupathdb/web-common` uses `isPrereleaseStudy` in numerous places, which you'll have to update when you upgrade `@veupathdb/web-study-data-access` there.

Reply:
Good point. I do think this is the time to address this. It would be great if we could get rid of all references to the user object.